### ISSUE TITLE
flutter master  branch - error G09587AA0: Struct 'Utf8' is empty. Empty structs are undefined behavior

### ISSUE THREAD

**Issue by micross** [bitsdojo/bitsdojo_window#28]
the depended plugins `ffi-0.1.3` and `win32-1.7.4+1` seem broken on flutter master branch, [flutter/flutter#76485](https://github.com/flutter/flutter/issues/76485)

```
Launching lib\main.dart on Windows in debug mode...
/C:/Users/micross/AppData/Local/Pub/Cache/hosted/mirrors.tuna.tsinghua.edu.cn%2547dart-pub/ffi-0.1.3/lib/src/utf8.dart(23,7): error G09587AA0: Struct 'Utf8' is empty. Empty structs are undefined behavior. [E:\workspace\peiyin\build\windows\flutter\flutter_assemble.vcxproj]
/C:/Users/micross/AppData/Local/Pub/Cache/hosted/mirrors.tuna.tsinghua.edu.cn%2547dart-pub/ffi-0.1.3/lib/src/utf16.dart(16,7): error G09587AA0: Struct 'Utf16' is empty. Empty structs are undefined behavior. [E:\workspace\peiyin\build\windows\flutter\flutter_assemble.vcxproj]
/C:/Users/micross/AppData/Local/Pub/Cache/hosted/mirrors.tuna.tsinghua.edu.cn%2547dart-pub/win32-1.7.4+1/lib/src/structs.dart(1111,7): error G09587AA0: Struct 'ENUMLOGFONTEX' is empty. Empty structs are undefined behavior. [E:\workspace\peiyin\build\windows\flutter\flutter_assemble.vcxproj]
/C:/Users/micross/AppData/Local/Pub/Cache/hosted/mirrors.tuna.tsinghua.edu.cn%2547dart-pub/win32-1.7.4+1/lib/src/structs.dart(2835,7): error G09587AA0: Struct 'BLUETOOTH_PIN_INFO' is empty. Empty structs are undefined behavior. [E:\workspace\peiyin\build\windows\flutter\flutter_assemble.vcxproj]
/C:/Users/micross/AppData/Local/Pub/Cache/hosted/mirrors.tuna.tsinghua.edu.cn%2547dart-pub/win32-1.7.4+1/lib/src/structs.dart(2960,7): error G09587AA0: Struct 'EXCEPINFO' is empty. Empty structs are undefined behavior. [E:\workspace\peiyin\build\windows\flutter\flutter_assemble.vcxproj]
/C:/Users/micross/AppData/Local/Pub/Cache/hosted/mirrors.tuna.tsinghua.edu.cn%2547dart-pub/win32-1.7.4+1/lib/src/structs.dart(2966,7): error G09587AA0: Struct 'PROPERTYKEY' is empty. Empty structs are undefined behavior. [E:\workspace\peiyin\build\windows\flutter\flutter_assemble.vcxproj]
/C:/Users/micross/AppData/Local/Pub/Cache/hosted/mirrors.tuna.tsinghua.edu.cn%2547dart-pub/win32-1.7.4+1/lib/src/structs.dart(2973,7): error G09587AA0: Struct 'PROPVARIANT' is empty. Empty structs are undefined behavior. [E:\workspace\peiyin\build\windows\flutter\flutter_assemble.vcxproj]
/C:/Users/micross/AppData/Local/Pub/Cache/hosted/mirrors.tuna.tsinghua.edu.cn%2547dart-pub/win32-1.7.4+1/lib/src/structs.dart(2978,7): error G09587AA0: Struct 'SAFEARRAY' is empty. Empty structs are undefined behavior. [E:\workspace\peiyin\build\windows\flutter\flutter_assemble.vcxproj]
/C:/Users/micross/AppData/Local/Pub/Cache/hosted/mirrors.tuna.tsinghua.edu.cn%2547dart-pub/win32-1.7.4+1/lib/src/structs.dart(2985,7): error G09587AA0: Struct 'CLSID' is empty. Empty structs are undefined behavior. [E:\workspace\peiyin\build\windows\flutter\flutter_assemble.vcxproj]
/C:/Users/micross/AppData/Local/Pub/Cache/hosted/mirrors.tuna.tsinghua.edu.cn%2547dart-pub/win32-1.7.4+1/lib/src/structs.dart(2992,7): error G09587AA0: Struct 'STATSTG' is empty. Empty structs are undefined behavior. [E:\workspace\peiyin\build\windows\flutter\flutter_assemble.vcxproj]
/C:/Users/micross/AppData/Local/Pub/Cache/hosted/mirrors.tuna.tsinghua.edu.cn%2547dart-pub/win32-1.7.4+1/lib/src/structs.dart(2999,7): error G09587AA0: Struct 'NLM_SIMULATED_PROFILE_INFO' is empty. Empty structs are undefined behavior. [E:\workspace\peiyin\build\windows\flutter\flutter_assemble.vcxproj]
/C:/Users/micross/AppData/Local/Pub/Cache/hosted/mirrors.tuna.tsinghua.edu.cn%2547dart-pub/ffi-0.1.3/lib/src/allocation.dart(47,33): error G733AAF94: Expected type 'T' to be a valid and instantiated subtype of 'NativeType'. [E:\workspace\peiyin\build\windows\flutter\flutter_assemble.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(238,5): error MSB8066: “E:\workspace\peiyin\build\windows\CMakeFiles\7c00e294f53b2c18e279f5a7262d1203\flutter_windows.dll.rule;E:\workspace\peiyin\build\windows\CMakeFiles\1641237fcbf9c44140010e4124d17fea\flutter_assemble.rule”的自定义生成已退 出，代码为 1。 [E:\workspace\peiyin\build\windows\flutter\flutter_assemble.vcxproj]
Building Windows application...
Exception: Build process failed.
```

**Comment by bitsdojo** [comment]
This is an issue with Flutter, not much I can do on my side:

https://stackoverflow.com/questions/65912637/how-can-i-fix-this-error-struct-utf8-is-empty-support-for-empty-structs-is-d

**Comment by micross** [comment]
update package:ffi to version 1.0.0  and package:win32 to preview version 2.0.0 may help.

**Comment by bitsdojo** [comment]
What is your Flutter version?

**Comment by bitsdojo** [comment]
I've just published 0.0.6

Please try it and let me know how that works.

**Comment by micross** [comment]
it works now, thank you.

```
Flutter 1.27.0-5.0.pre.112 • channel master • https://github.com/flutter/flutter.git
Framework • revision 90c66f5db3 (2 hours ago) • 2021-02-22 18:21:03 -0800
Engine • revision 07afcf7d0f
Tools • Dart 2.13.0 (build 2.13.0-52.0.dev)
```
